pubpub/pubpub
# prevent empty wokflow email targets

addresses #2049 along with manually transferring old `targetEmailAddress` to the mistaken `[NULL]`-valued cells for `targetEmailAddresses`

_test plan_

1. edit workflow for any SubmissionWorkflow
2. head to correspondence tab of workflow editor
3. attempt to save with empty target email address field (should be prevented)
4. save with one target address
5. save again with multiple
